utilities/src/main/java/io/syndesis/qe/bdd/validation/GoogleCalendarValidationSteps.java

```diff
@@ -0,0 +1,33 @@
+package io.syndesis.qe.bdd.validation;
+
+import com.google.api.services.calendar.model.Calendar;
+import com.google.api.services.calendar.model.Event;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import io.syndesis.qe.utils.GoogleCalendarUtils;
+import org.junit.Assert;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+
+public class GoogleCalendarValidationSteps {
+    @Autowired
+    private GoogleCalendarUtils gcu;
+
+    @Then("verify that event \"([^\"]*)\" with description \"([^\"]*)\" exists in calendar \"([^\"]*)\" using account \"([^\"]*)\"")
+    public void verifyEventExists(String eventSummary, String eventDescription, String calendarName, String googleAccount) throws IOException {
```

Comment:
Two problems with this and the following methods:
1. copy pasta
2. the method names don't reflect what's going on